1.0/profile.ttl

```diff
@@ -0,0 +1,64 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    owl:versionIRI <http://www.opengis.net/def/geosparql/1.0> ;   
+    dcterms:created "2020-11-20"^^xsd:date ;
+    dcterms:modified "2020-12-19"^^xsd:date ;
+    dcterms:issued "2021"^^xsd:gYear ;
+    dcterms:publisher <https://www.ogc.org> ;
+    dcterms:title "GeoSPARQL 1.1 Profile" ;
+    dcterms:description """This is a 'profile declaration' for the GeoSPARQL 1.0 specification (standard) as originally published in 2012. It describes the multiple parts of the specification and how the standard relates to other standards. It is formulated according to the Profiles Vocabulary."""@en ;
+    skos:scopeNote """Profile declarations are used to allow data to identify specifications in a way that allows data to make conformance claims to them - a broader conformance claim than that made to individual 'conformance classes' that are commonly found in recent OGC specifications. Profile delcarations are also used to describe and relate the often multiple parts of specifications: ontologies, specification 'documents', vocabularie, validators and so on. This declaration describes where those parts are, what form (format) they take, what information models they implement and what roles they play."""@en ;    
```

Comment:
Two typos: delcarations and vocabularie

Reply:
Fixed in Commit https://github.com/opengeospatial/ogc-geosparql/pull/58/commits/58677653615b875e834f6eb1661933c50dadcf59